modules/nf-core/spring/decompress/main.nf

```diff
@@ -21,7 +21,7 @@ process SPRING_DECOMPRESS {
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
     def VERSION = '1.1.1' // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
-    def output = meta.single_end ? "-o ${prefix}.fastq.gz" : "-o ${prefix}_R1.fastq.gz ${prefix}_R2.fastq.gz"
+    def output = task.ext.one_direction ? "-o ${prefix}.fastq.gz" : "-o ${prefix}_R1.fastq.gz ${prefix}_R2.fastq.gz"
```

Comment:
what is wrong with using `meta.single_end` ?

Reply:
Technically, we could get this working by setting `meta.single_end` to `true` in the input-channel (and then I would suggest removing it from the meta of the output-channel on the receiving side so as to avoid any confusion downstream), but since the sample isn't `single_end`, so I would find it confusing. The input-channel contains R1.fastq.gz.spring or R2.fastq.gz.spring. Maxime suggested `meta.one_strand` which I find more appropriate than `meta.single_end`.

Reply:
ok, we should just not introduce any new `task.ext` here